webview/src/plots/components/ZoomablePlotWrapper.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { ReactElement, PropsWithChildren } from 'react'
+import Tooltip from '../../shared/components/tooltip/Tooltip'
+
+interface ZoomablePlotWrapperProps {
+  title?: string
+  id: string
+}
+
+export const ZoomablePlotWrapper: React.FC<
+  PropsWithChildren<ZoomablePlotWrapperProps>
+> = ({ title, id, children }) => {
+  const isTitleCut = title?.indexOf('…') === 0
```

Comment:
TIL we aren't doing this in code.

Reply:
no wait.... we are doing this in code but you can bypass because the id is the title, got it